sdk/src/vote_program.rs

```diff
@@ -37,6 +38,32 @@ impl Vote {
     }
 }
 
+#[derive(Serialize, Default, Deserialize, Debug, PartialEq, Eq, Clone)]
+pub struct Lockout {
+    pub slot_height: u64,
+    pub confirmation_count: u32,
+}
+
+impl Lockout {
+    pub fn new(vote: &Vote) -> Self {
+        Self {
+            slot_height: vote.slot_height,
+            confirmation_count: 1,
+        }
+    }
+
+    // The number of slots for which this vote is locked
+    pub fn lockout(&self) -> u64 {
+        (INITIAL_LOCKOUT as u64).pow(self.confirmation_count)
+    }
+
+    // The slot height at which this vote expires (cannot vote for any slot
+    // less than this)
+    pub fn lock_height(&self) -> u64 {
```

Comment:
How about `expiration_slot_height()`? where "expiration" is whatever is decided above.